NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 9view all categories

Description
[Trello Ticket #9](https://trello.com/c/fTBUUbwT)

Added functionality to view a list of the logged-in user's category list. The list is sorted in order of the category name in A-Z order.

Testing Instructions
1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout 9viewAllCategories
5. If you don't see the most recent version of my files, git pull origin 9viewAllCategories
6. If you haven't created the Tabloid database, please do so.
7. Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Click the execute with debugger solid green button to launch the app
10. If you need to login use leo@gmail.com
12. From the home page, click the “Category Management” option in the navbar
13. This should take you to a page with the list of categories.

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
